themes/inbeat/layouts/partials/site-footer.html

```diff
@@ -195,7 +195,7 @@
                     <div class="content">
                         <ul>
                             {{- range .Site.Data.navigation.footer.support -}} 
```

Comment:
It doesn't make sense to keep this as an array (map in Hugo terms) if we hard code the URL. Change to a normal string field, or make the url dynamic inside the map.